scripts/validate-changelog.py

```diff
@@ -22,7 +22,7 @@
         skip_changelog = True
         break
 
-  matches = re.search(r'<changelog>(.+)</changelog>', pr_description) is not None
+  matches = re.search(r'<changelog>(.+)</changelog>', pr_description, flags=re.S) is not None
```

Comment:
Is this related to OP or some cleanup? 🤔

Reply:
Unrelated, allows for changelog message tag with new lines.